### PR TITLE
Add new messaging to Equipment Manager

### PR DESCRIPTION
**Pull request by jayRyan24**
New gloves don't have messaging. Locks up on repair and swapping. 

a pair of ridged leather gauntlets with silver-wrapped cuffs

### CODE REVIEW

File: equipmanager.lic [hunk 1]
```diff
@@ -212,7 +212,7 @@ class EquipmentManager
 
     todo.all? do |held_item|
       if info = worn_items.find { |item| item.short_regex =~ held_item }
-        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop')
+        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop', 'slide effortlessly onto your')
```
Comment (jayRyan24) [inline equipmanager.lic]: Will this work? Seems to not be?

Comment (rpherbig) [inline equipmanager.lic]: This looks reasonable. Is it not working?

Comment (jayRyan24) [inline equipmanager.lic]: Yeah, still locks up. Let me test more tonight and I'll see. It was checking something shortened. Forget the exact, but I'll get it tested tonight

**Comment by jayRyan24** [comment]
Updated to address issue #1745.

**Comment by rpherbig** [comment]
Is this good to merge?

**Comment by jayRyan24** [comment]
@rpherbig This is ready for merging.

**Pull request merged**